### ISSUE TITLE
Use Shizuku to override new Android security patch

### ISSUE THREAD

**Issue by neobenedict** [rayshift/translatefgo#76]
Fix error "To protect your privacy, choose another folder." and related errors by using Shizuku

 https://twitter.com/MishaalRahman/status/1719416906174681554

**Comment by neobenedict** [comment]
Well, strangely, I can't even reproduce the actual issue on an up-to-date pixel 5 running android 14, which makes it hard to fix it. :/

**Comment by Perseo2499** [comment]
Hi, I've bee facing the same issue on my device with the latest update to A14. I'm a root user, so there is a solution that worked for me using a Lsposed module. 

https://github.com/DanGLES3/NoStorageRestrict

Download the module, activate it on Lsposed app and reboot the device. The storage permission block will vanish :)

**Comment by neobenedict** [comment]
https://github.com/topjohnwu/libsu/issues/127#issuecomment-1180228500 + https://www.xda-developers.com/implementing-shizuku/ AIDL

lol @ getting this to work with xamarin...

**Comment by neobenedict** [comment]
```02-18 01:46:49.351  6274  6274 I TranslateFGO: Starting NGFS service with context=android.app.ContextImpl@bf907b3
02-18 01:46:49.821  6181  6181 I TranslateFGO: Bound NGFS.
02-18 01:46:49.822  6274  6294 I TranslateFGO: pid=6274, uid=2000
02-18 01:46:49.822  6274  6294 I TranslateFGO: Listing files.
02-18 01:46:49.822  6274  6294 I TranslateFGO: file: 969b46577f365fadeb79ef14cf5d6370
02-18 01:46:49.822  6274  6294 I TranslateFGO: file: 54cc790bf952ea710ed7e8be08049531
02-18 01:46:49.822  6274  6294 I TranslateFGO: file: bfaa
02-18 01:46:49.822  6274  6294 I TranslateFGO: file: d713```

Good news, we're getting somewhere.

**Comment by Perseo2499** [comment]
Very excited to hear this. Can't wait to see the results!!!

**Comment by neobenedict** [comment]
OK, here is a beta release: http://apk.rayshift.io/translatefgo/io.rayshift.translatefgo-22007_3.apk

Could you test it and let me know your feedback, if it works or not, if you get any errors please screenshot them and post here.

**Comment by Perseo2499** [comment]
Worked for me. 

I disable the modules that had it working and cleaned the data from the app. 

As you can see, I had again the storage permission issue. I used the Shizuka method and it worked fine. It didn't ask me for the folder of the game, It went to the installation menu directly. 

And as you can see in the picture of the games, the translations work :) 
![Screenshot_20240219-090603_Files](https://github.com/rayshift/translatefgo/assets/53657268/8306e848-15be-400b-81fc-bf230d4cd45d)
![Screenshot_20240219-090629_Translate Fate_GO](https://github.com/rayshift/translatefgo/assets/53657268/b3ef12c9-6628-4c9c-ab19-e13b6197af04)
![Screenshot_20240219-090634_Translate Fate_GO](https://github.com/rayshift/translatefgo/assets/53657268/ea167974-c696-477e-bad5-cacb32bf7db7)
![Screenshot_20240219-090749_Translate Fate_GO](https://github.com/rayshift/translatefgo/assets/53657268/6a20a502-50a1-40c6-93d8-2ccd7d5ffe30)
![Screenshot_20240219-090756_Translate Fate_GO](https://github.com/rayshift/translatefgo/assets/53657268/421fd949-4b45-4acd-938f-6bdcec6f0be9)
![Screenshot_20240219-091111_Fate_GO](https://github.com/rayshift/translatefgo/assets/53657268/e52ecb09-de1c-45c2-8d31-177781c09fd9)

**Comment by Perseo2499** [comment]
I forgot to mention it. I was running Shizuku with root, so I stopped it and enabled by wireless debugging. I also worked :)

**Comment by neobenedict** [comment]
Fixed with version 2.2.0.

**Comment by racsoemzdaidomefenix-ux** [comment]
Buenas me presenta esta falla no me muestra la parte para instalar y traducir soy android 13 dailn neo el año pasado me corría bien enero me lo tradujo pero a partir de Andrómeda no me lanza para actualizarlo no me deja pasar la captura porque dice falla en la imagen disculpa y buenas noche
